### PR TITLE
FSET-1302: Print assessment scores and feedback

### PR DESCRIPTION
**Pull request by lukastymo**

### CODE REVIEW

File: app/models/SecurityUser.scala [hunk 1]
```diff
@@ -37,7 +37,9 @@ case class CachedUser(
   email: String,
   isActive: Boolean,
   lockStatus: String
-)
+) {
+  def fullName = s"$firstName $lastName"
```
Comment (henricook) [inline app/models/SecurityUser.scala]: Could probably be a val? Not that i think it'll hold this PR up - miniscule efficiency different

File: app/views/application/assessmentCentreTestScoresAndFeedback.scala.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @(scores: connectors.exchange.CandidateScoresAndFeedback, competencyAverageResult: connectors.exchange.CompetencyAverageResult)(implicit request: Request[_], flash: Flash, user: Option[CachedData], feedbackUrl: String, messages: Messages)
 
-@main_template(title = "Your scores and feedback | Apply for a Civil Service apprenticeship") {
```
Comment (henricook) [inline app/views/application/assessmentCentreTestScoresAndFeedback.scala.html:3]: Does this affect acceptance tests?

File: app/views/home/dashboard.scala.html [hunk 3]
```diff
@@ -31,7 +31,7 @@
 }
 
 @renderFullName(user: CachedData) = {
-@user.user.firstName @user.user.lastName
+    user.user.fullName
```
Comment (henricook) [inline app/views/home/dashboard.scala.html]: Doesn't need an @?

File: app/views/application/assessmentCentreTestScoresAndFeedback.scala.html (hunk 2 above)
Comment (lukastymo) [reply app/views/application/assessmentCentreTestScoresAndFeedback.scala.html:3]: That was a typo. The right side is added automatically. Previously it was duplicated: <title> Your scores and feedback | Apply for a Civil Service apprenticeship | Apply for a Civil Service apprenticeship

File: app/views/home/dashboard.scala.html (hunk 3 above)
Comment (lukastymo) [reply app/views/home/dashboard.scala.html]: It does

File: app/models/SecurityUser.scala (hunk 1 above)
Comment (lukastymo) [reply app/models/SecurityUser.scala]: Done

**Pull request merged**